src/services/transform.ts

```diff
@@ -20,6 +21,16 @@ export default (config: ConfigManager): TransformService => {
             }
             return response.text();
         },
+        exportTransform: async (xml: string) => {
+            let response;
+            try {
+                console.log('Transforming exported XML');
```

Comment:
Did you intend to leave this debug in?

Reply:
Yep I leave these in to log that transforming happened and to help debug any error thrown during transform. (The same as on import)